package.json

```diff
@@ -55,11 +55,15 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router": "^6.0.0",
+    "react-router-dom": "^6.3.0",
     "rollup": "^2.51.1",
     "rollup-plugin-size": "^0.2.2",
     "rollup-plugin-terser": "^7.0.2",
     "typescript": "^4.3.2"
   },
+  "resolutions": {
+    "@types/react": "^18.0.0"
+  },
```

Comment:
Why is resolutions needed here?

Reply:
A conflict in types between react and react-dom. Resolutions resolves this which is preferred over skipLibChecks flag.